### PR TITLE
Adding the Hathi logo to the home page

### PR DESCRIPTION
**Pull request by carolyncole**
also added an alert icon to the home page
change hathi_ap_url to hathi_api_url also

### CODE REVIEW

File: app/views/catalog/_home_text.html.erb [hunk 1]
```diff
@@ -14,9 +14,16 @@
...
-      Look for the "Temporary Digital Access from Hathi Trust" link in the "Available Online" section.
-  </div>
+  <card class="card card-body bg-light">
+      <card-media>
+        <%= link_to "https://www.hathitrust.org/covid-19-response" do %>
+          <%= image_tag("250px-HathiTrust_logo.svg.png") %>
+        <% end %>
+      </card-media>
+      <card-content>    
+        Many print items are now available online from Hathi Trust.
```
Comment (christinach) [inline app/views/catalog/_home_text.html.erb:24]: can we change the icon-warning to somtheing else? maybe icon info?

**Comment by christinach** [comment]
@carolyncole looks great. Can we fix the coverage if it's still failing? It failed earlier.

**Comment by carolyncole** [comment]
@christinach I think it was just failing because the test runs were terminating early. Looks ok now...  Just waiting on 2_6

**Comment by christinach** [comment]
yes it looks green now!

**Pull request merged**